src/io/ffmpeg_io.ml

```diff
@@ -153,6 +157,19 @@ let () =
         args ~t:Lang.int_t "int";
         args ~t:Lang.float_t "float";
         args ~t:Lang.string_t "string";
+        ( "self_sync",
+          Lang.bool_t,
+          Some (Lang.bool true),
+          Some
+            "Set to `true` if the input is a real-time stream, i.e. icecast, \
+             SRT, etc. and `false` is not, i.e. local file, etc.." );
+        ( "clock_safe",
+          Lang.bool_t,
+          Some (Lang.bool false),
+          Some
+            "Set to `true` if the source needs to be attached to its own \
+             clock. Should be set to `true` when `self_sync` is set to `true` \
+             in most cases." );
```

Comment:
Do we really need this parameter since it seems to be implied by `self_sync`? At least I would put a `Lang.nullable_t Lang.bool_t` so that the default value is the same as `self_sync`.